### ISSUE TITLE
Getting "module Timex.DateTime is not available" error after upgraded Timex to version 3.0.3

### ISSUE THREAD

**Issue by seymores** [c-rack/quantum-elixir#110]
```
[error] Task #PID<0.1262.0> started from Quantum terminating
** (UndefinedFunctionError) function Timex.DateTime.from/1 is undefined (module Timex.DateTime is not available)
    Timex.DateTime.from({{2016, 7, 16}, {5, 48, 0}})
    (quantum) lib/quantum/executor.ex:18: Quantum.Executor.convert_to_timezone/2
    (quantum) lib/quantum/executor.ex:84: Quantum.Executor.execute/2
    (elixir) lib/task/supervised.ex:94: Task.Supervised.do_apply/2
    (elixir) lib/task/supervised.ex:45: Task.Supervised.reply/5
    (stdlib) proc_lib.erl:240: :proc_lib.init_p_do_apply/3
Function: &Quantum.Executor.execute/2
```

**Comment by c-rack** [comment]
Thanks for reporting! We need to upgrade quantum to Elixir >= 1.3 and use the new date/time types. Then, we would release a 2.0 version of quantum, indicating that this version bump breaks backwards compatibility (i.e. pre-elixir-1.3 support). Any pull requests in that directions are welcome.

**Comment by c-rack** [comment]
Fixed by #111
